usaspending_api/references/models.py

```diff
@@ -401,10 +474,362 @@ class LegalEntity(DataSourceTrackedModel):
     individual = models.TextField(blank=True, null=True)
 
     def save(self, *args, **kwargs):
+        LegalEntity.update_business_type_categories(self)
         super(LegalEntity, self).save(*args, **kwargs)
 
         LegalEntityOfficers.objects.get_or_create(legal_entity=self)
 
+    @staticmethod
+    def update_business_type_categories(le):
```

Comment:
Good call on putting this logic into the model instead of one of our separate helper files.